src/Native/Runtime/MathHelpers.cpp

```diff
@@ -82,6 +82,70 @@ EXTERN_C REDHAWK_API UInt64 REDHAWK_CALLCONV RhpDbl2ULng(double val)
     return((UInt64)val);
 }
 
+// CORERT Specific - on Project N the arguments to these helpers are inverted
+#ifdef CORERT
+#include <cmath>
+
+#define CLR_NAN_32 0xFFC00000
+#define CLR_NAN_64 0xFFF8000000000000L
```

Comment:
OK, seems like `-nan(0)` and `-nanf(0)` produce the same byte sequence, so I will use that. I won't pretend I know about floating point and why `nan`/`nanf` are functions that take parameters, so scream if this is wrong.